week7/matchTheCard/src/components/CardSection.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import Card from "./Card";
+import cat from "../assets/cat";
+import { useRecoilValue } from "recoil";
+import {
+  totalCards,
+  currentCards,
+  matchedCards,
+} from "../recoil/atom/cardArrays";
+import React from "react";
+
+interface CardSectionProps {
+  handleCardClick: (index: number) => void;
+}
+
+function CardSection({ handleCardClick }: CardSectionProps) {
+  const matchedCard: number[] = useRecoilValue(matchedCards);
+  const currentCard: number[] = useRecoilValue(currentCards);
+  const totalCard: string[] = useRecoilValue(totalCards);
+
+  if (cat) {
+    return (
+      <section>
+        <Cards>
+          {totalCard.map((name: string, index: number) => (
+            <Card
+              key={index}
+              imageSrc={cat.find((v) => v.name === name)?.src}
```

Comment:
쿄쿄쿄ㅎㅎ 위에 if (cat) 빼구 요렇게 옵셔널 체이닝으루 해도 되게땅!!
```suggestion
  imageSrc={cat?.find((v) => v.name === name)?.src}
```

그리구 밑에 else 구문도 다 빼도 될거가튼데!!??